src/dmap_import/util_rds.py

```diff
@@ -56,6 +77,60 @@ def get_db_password() -> str:
     return db_password
 
 
+def create_db_connection_string() -> str:
+    """
+    produce database connection string from environment
+    """
```

Comment:
Broke this out into it's own function because `psql` command required connection string to operate.